source/4_tools/console_app/console_app.sharpmake.cs

```diff
@@ -22,6 +22,13 @@ protected override void SetupConfigSettings(RexConfiguration conf, RexTarget tar
     conf.Options.Add(Options.Vc.Linker.SubSystem.Console);
   }
 
+  protected override void SetupOutputType(RexConfiguration conf, RexTarget target)
+  {
+    base.SetupOutputType(conf, target);
+
+    conf.Output = Configuration.OutputType.Exe;
```

Comment:
INFO: Is this not the default output type of an application?

Reply:
yes, I just set it for readability

Reply:
I think that would be implied from an "app" but ok